doc/nrf/protocols/wifi/sniffer_rx_operation.rst

```diff
@@ -0,0 +1,136 @@
+
+.. _ug_nrf70_developing_monitor_mode_receive_packet_metadata:
+
+Monitor mode receive packet metadata
+************************************
+
+The IEEE 802.11 packet that is captured by the nRF70 Series device will prepend a proprietary header ``raw_rx_pkt_header`` which contains the following information about the received 802.11 packet.
+
+.. list-table:: Wi-Fi packet reception header elements
+   :header-rows: 1
+
+   * - Parameter name
+     - Type
+     - Description
+   * - frequency
+     - unsigned short
+     - Provides the current frequency in which the packet was received.
+       The frequecy element can be used to compute the packet receive channel.
+   * - signal
+     - signed short
+     - Signal strength of the received packet
+   * - rate_flags
+     - unsigned char
+     - Provides data on whether the received 802.11 packet is a legacy, HT, VHT or a HE packet
+   * - rate
+     - unsigned char
+     - Provides the data rate at which the packet was received.
+       It can be a ``legacy rate`` or an ``MCS rate`` based on the ``rate_flags`` value.
+
+You can refer to the relevant structure at:
+
+:file:`nrfxlib/nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h` - for the raw RX header
+
+.. _ug_nrf70_developing_monitor_mode_receive_operation:
+
+Monitor mode receive operation
+******************************
+
+An IEEE 802.11 packet captured by the nRF70 Series device in Monitor mode will be sent up to the Wi-Fi driver which will append a radio information header to the received packet and present the resulting set to the user application.
+To receive the captured traffic in Monitor mode, the user application layer needs to open a raw socket to the Wi-Fi driver.
+
+The following figure illustrates the packet structure and IEEE 802.11 packet sniffer operational flow:
+
+.. figure:: images/nrf7000_packet_sniffer_operation.png
+   :alt: IEEE 802.11 packet sniffer operational flow
+
+.. _ug_nrf70_developing_monitor_mode_receive_operation_with_tx_injection:
+
+Monitor mode receive operation in conjunction with TX injection
+***************************************************************
+
+TX injection mode can be enabled for operation when the nRF70 Series device is configured to operate in Monitor mode.
+
+TX injection mode allows the transmission of a raw 802.11 transmit packet.
+Raw IEEE 802.11 packets are packets that are not modified by the 802.11 Medium Access Control (MAC) layer during transmission by the nRF70 Series device.
+
+TX injection mode can be enabled by invoking the API ``net_eth_txinjection_mode``.
+The API has the following parameters as described below.
+
+.. list-table:: TX injection mode API
+   :header-rows: 1
+
+   * - Parameter name
+     - Type
+     - Description
+   * - iface
+     - struct net_if
+     - Network Interface structure
+   * - enable
+     - boolean
+     - Parameter to enable/disable TX injection mode.
+       ``1`` enables TX injection and ``0`` disables TX injection.
+
+.. note::
```

Comment:
I suggest moving this note to after line 116 - 

It also depends on, what is the order of steps to be able to enable TX injection mode?
Is it: 
1. Enable (kconfig option)
2. Invoke (API)  

And both steps are required? If so, I would describe it in that order (or which ever order is necessary), and not present it as a note, rather an instruction in this section.